src/lib/http.coffee

```diff
@@ -21,11 +23,30 @@ module.exports = (dnschain) ->
             @log = gNewLogger 'HTTP'
             @log.debug "Loading HTTPServer..."
             @rateLimiting = gConf.get 'rateLimiting:http'
+            app = express()
+            router = express.Router()
+
+            router.route(/\/v1\/(\w+)\/(\w+)(\/([^\/]+)(\/(\w+)(\.(\w+))?)?)?/)
+            .get (req, res) =>
+                [chain, resource, _, property, _, operation, _, fmt] = req.params
+                @log.debug gLineInfo('GET v1 API'),
+                    {chain:chain, resource:resource, property:property, operation:operation, fmt:fmt}
+                if chain is 'resolver'
+                    # TODO do resolver stuff
+                    res.end()
+                else
+                    return @notFound(res,400,'No Blockchain Found',chain,->) if not (resolver=@dnschain.chains[chain])
+                    return @notFound(res,400,'No Resource Found',resource,->) if not (resolveResource=resolver.resources[resource])
+                    resolveResource.call resolver, property, operation, fmt, req.query, @postResolveCallback(res, property,->)
 
+            # support old api
+            router.get '/*', @callback.bind(@)
+
+            app.use(router)
```

Comment:
Actually, looking more at the app.use docs, it seems you might want to do more like so:

```coffeescript
app.use "/v1", opennameRoute
app.use "/", oldRoute
```